email.yml

```diff
@@ -6,6 +6,28 @@ servers:
   - url: http://localhost:8082
     description: Generated server url
 paths:
+  /api/v1/autotransport/emailservice/reminderMails:
```

Comment:
we should never use camel case in resource name.
convention wants to use hyphen but we never did. so just use lower case.